docs/publishing-artifacts.md

```diff
@@ -0,0 +1,63 @@
+# Publishing to Sonatype
+
+## Prerequisites
+
+* Need to have `pgp` installed. For MacOS, do a `brew install pgp`. In Ubuntu you can do `sudo apt install gnupg`. Or install from [GnuPG](https://www.gnupg.org/download/). Check the following after install:
+
+```
+$ gpg --version
+gpg (GnuPG) 2.2.17
+libgcrypt 1.8.5
+Copyright (C) 2019 Free Software Foundation, Inc.
+License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>
+This is free software: you are free to change and redistribute it.
+There is NO WARRANTY, to the extent permitted by law.
+
+Home: /Users/debasishghosh/.gnupg
+Supported algorithms:
+Pubkey: RSA, ELG, DSA, ECDH, ECDSA, EDDSA
+Cipher: IDEA, 3DES, CAST5, BLOWFISH, AES, AES192, AES256, TWOFISH,
+        CAMELLIA128, CAMELLIA192, CAMELLIA256
+Hash: SHA1, RIPEMD160, SHA256, SHA384, SHA512, SHA224
+Compression: Uncompressed, ZIP, ZLIB, BZIP2
+```
+
+* Once you have `pgp` installed generate a key pair and distribute the keys to public key servers. Follow the instructions in this [section](https://www.scala-sbt.org/1.x/docs/Using-Sonatype.html#step+1%3A+PGP+Signatures) of the sbt document.
+
+* Create a file `gpg.sbt` under `~/.sbt/1.0/plugins` containing the following line:
+
+```
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.0")
+```
+
+* Create a file named `sonatype.credentials` under `~/.sbt` containing your Sonatype credentials:
+
+```
```

Comment:
I would suggest just showing what must be in the file, not a pwd and cat, since you already mentioned above where the file must be?